godel_surface_detection/src/detection/surface_detection.cpp

```diff
@@ -764,39 +772,6 @@ bool SurfaceDetection::apply_kdtree_radius_search(const Cloud& query_points,cons
 
 }
 
-bool SurfaceDetection::apply_fast_triangulation(const Cloud& in,
-		const Normals& normals,
-		pcl::PolygonMesh& mesh)
-{
-	// search tree
-	pcl::search::KdTree<pcl::PointNormal>::Ptr tree (
-			new pcl::search::KdTree<pcl::PointNormal>);
-
-	// cloud with normals
-	pcl::PointCloud<pcl::PointNormal>::Ptr cloud_with_normals(
-			new pcl::PointCloud<pcl::PointNormal>());
-	pcl::concatenateFields(in,normals,*cloud_with_normals);
-	tree->setInputCloud(cloud_with_normals);
-
-	// triangulation
-	pcl::GreedyProjectionTriangulation<pcl::PointNormal> gt;
-	gt.setConsistentVertexOrdering(true);
-	gt.setSearchRadius(params_.tr_search_radius);
-	gt.setMu(params_.tr_mu);
-	gt.setMaximumNearestNeighbors(params_.tr_max_nearest_neighbors);
-	gt.setMaximumSurfaceAngle(params_.tr_max_surface_angle);
-	gt.setMinimumAngle(params_.tr_min_angle);
-	gt.setMaximumAngle(params_.tr_max_angle);
```

Comment:
There are some members that are only used by this fast triangulation routine ("tr_" prefix).  Since they are no longer being used they should be removed.  However this isn't a trivial task because some gui components in godel_plugins assume that they exists.  Thus this should probably be addressed in a separate PR.